SPECS/kubevirt/kubevirt.spec

```diff
@@ -15,27 +15,27 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
+%define _missing_build_ids_terminate_build 0
```

Comment:
Where is this variable being used?

Reply:
i think related to build errors like this:

`Missing build-id in /usr/src/azl/BUILDROOT/kubevirt-1.2.0-1.azl3.x86_64/usr/bin/virtctl`